### ISSUE TITLE
Cant skip default dns in vcd_network_isolated

### ISSUE THREAD

**Issue by ProFfeSsoRr** [terraform-providers/terraform-provider-vcd#434]
I not need DNS servers in my networks, but can't skip default dns1 8.8.8.8 and dns2 8.8.4.4 values.

### Terraform Version
Terraform v0.12.18
+ provider.vcd (2.6.0)

### Affected Resource(s)
Please list the resources as a list, for example:
- vdc_network_isolated

### Terraform Configuration Files
```hcl
resource "vcd_network_isolated" "stage" {
  name    = "stage"
  gateway = "10.1.0.1"
  netmask = "255.255.255.0"
  dns1 = null
  dns2 = null
}
```

### Expected Behavior
Not need to change DNS servers at all.

### Actual Behavior
```
Terraform will perform the following actions:

  # vcd_network_isolated.stage must be replaced
-/+ resource "vcd_network_isolated" "stage" {
      ~ dns1    = "" -> "8.8.8.8" # forces replacement
      ~ dns2    = "" -> "8.8.4.4" # forces replacement
```

**Comment by dataclouder** [comment]
Thanks for reporting this issue.
You are right. These defaults are probably too specific.
As a workaround, can you use your gateway IP for both DNS fields?

```hcl
resource "vcd_network_isolated" "stage" {
  name    = "stage"
  gateway = "10.1.0.1"
  netmask = "255.255.255.0"
  dns1    = "10.1.0.1"
  dns2    = "10.1.0.1"
}
```

**Comment by ProFfeSsoRr** [comment]
As workaround it works, thanks. I think default value for dns fileds will be empty because network without this setting at all is normal.

**Comment by dataclouder** [comment]
Fixed in PR #465
